Adding additional classes to reveal-overlay element

jefsnare opened the pull request:
As requested in #9822. Adding additional classes to `reveal-overlay`
element.

Not the possibility to replace the class, since many styling and some JS
handles react on this class.

Commit:
Reveal Overlay additionalOverlayClassNames

As requested in #9822. Adding additional classes to `reveal-overlay`
element.

Not the possibility to replace the class, since many styling and some JS
handles react on this class.

Commit:
Restore default value for font-family

Owlbertz commented:
I see that having multiple classes on the overlay might be useful in some cases if you want to style multiple overlays differently.
Would you be able to write a short test case for your feature that checks if classes are actually added?

jefsnare commented:
@Owlbertz Yes ofcourse, I'll add them asap

Commit:
Added test for additional reveal classes

While writing the test I've also refactored the additionalOverlayClasses
check.

jefsnare commented:
@Owlbertz Test added, and minor refactor done.

kball commented:
Awesome. Looks good to me!
